DSCResources/cMsmqQueue/cMsmqQueue.psm1

```diff
@@ -207,6 +218,11 @@ function Set-TargetResource
         [String]
         $Name,
 
+        [Parameter(Mandatory = $false)] # not mandatory for backward compatibility (even if it prevents to have same name for public & private queue)
```

Comment:
I think this comment is not really needed. But if you insist on leaving it, please place it on a separate line.